.github/workflows/ci-build-release-napi.yml

```diff
@@ -56,31 +62,19 @@ jobs:
         with:
           python-version: ${{ matrix.python }}
           
-      - name: Cache Dependencies
-        id: cache-dependencies
-        uses: actions/cache@v3
-        with:
-          path: pkg/mac/build
-          key: ${{ runner.os }}-${{ matrix.arch }}-mac-${{ hashFiles('pkg/mac/build-cpp-deps-lib.sh') }}
-
```

Comment:
Why do we remove this cache?

Reply:
Because we don't need to compile the cpp client, just need to download it, and it will very fast.

Reply:
Get it. Thanks. That helps a lot!